packages/medusa/src/services/order-edit.ts

```diff
@@ -388,6 +411,85 @@ export default class OrderEditService extends TransactionBaseService {
     return orderEdit
   }
 
+  async addLineItem(orderEditId: string, data: AddOrderEditLineItemInput) {
```

Comment:
suggestion: I think that once my pr will be merged, this method will be able to just call [that](https://github.com/medusajs/medusa/pull/2246/files#diff-a26e080c56da65f0067681a51f853510451acbbe4700216d78a333826a71357dR530) and then create the item change wdyt?